### PR TITLE
fix default css filename (#259)

### PR DESCRIPTION
**Pull request by dinhlongviolin1**
Fix bug #259 cssfile name for FJ

### CODE REVIEW

File: taipy/gui/gui.py [hunk 1]
```diff
@@ -25,8 +25,8 @@
 
 import __main__
 import markdown as md_lib
-from flask import Blueprint, Flask, request, send_from_directory
 import tzlocal
+from flask import Blueprint, Flask, request, send_from_directory
```
Comment (FabienLelaquais) [inline taipy/gui/gui.py:29]: How come these were added?

**Review by FabienLelaquais — APPROVED** [review]
👍

**Review by FredLL-Avaiga — APPROVED** [review]
+1

File: taipy/gui/gui.py (hunk 1 above)
Comment (dinhlongviolin1) [reply taipy/gui/gui.py:29]: These were here from the beginning. Pre-commit just formatted it and move it to another line

**Pull request merged**